PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterNonLinearity.h

```diff
@@ -43,6 +43,8 @@ class AliEmcalCorrectionClusterNonLinearity : public AliEmcalCorrectionComponent
   TH1F                  *fEnergyDistAfter;           //!<!energy distribution after
   TH2F                  *fEnergyTimeHistAfter;       //!<!energy/time distribution after
   
+  Bool_t                 fSetForceClusterE;          ///< // Only for backwards compatibility, force cluster->E() to be set to the cluster non-linearity corrected energy. Off by default. For the standard methods, see: http://alidoc.cern.ch/AliPhysics/master/READMEcontainers.html#emcalContainerClusterEnergyCorrections
```

Comment:
This is super trivial, but this has a double comment marker (`///< //`) - could you please remove the `//`? I know this is trivial, but I'd like to keep things as clean as possible